### ISSUE TITLE
SharedArrayBuffer

### ISSUE THREAD

**Issue by DerThorsten** [DerThorsten/jupyterlite_xeus_lua#8]
@jtpio  here a few infos on sharedarraybuffers.

https://developers.google.com/search/blog/2021/03/sharedarraybuffer-notes
https://web.dev/cross-origin-isolation-guide/

Not sure what the cross-origin isolation thingy actually means and if we could use it to keep the SharedArrayBuffer

**Comment by jtpio** [comment]
So with a `serve.py` as follows:

```python
from http import server


class MyHTTPRequestHandler(server.SimpleHTTPRequestHandler):
    def end_headers(self):
        self.send_header("Cross-Origin-Opener-Policy", "same-origin")
        self.send_header("Cross-Origin-Embedder-Policy", "require-corp")
        server.SimpleHTTPRequestHandler.end_headers(self)


if __name__ == '__main__':
    server.test(HandlerClass=MyHTTPRequestHandler)

```

And starting the HTTP server with `python serve.py`, we indeed seem to get past this.

**Comment by DerThorsten** [comment]
@jtpio  but do we have control over this when we host the app somewhere like gh. pages?

**Comment by jtpio** [comment]
Looks like this was fixed by #14 

Working nicely locally :tada: without any extra flag
